bug: gate proxy disconnects players after sending a player a `Entity Sound Effect` packet

Code used:

Bare bones gate (Extreme simplification of my existing gate implementation without any plugins or anything)
```go
func main() {
	gate.Execute()
}
```

Very simple Minestom server
```java
public class Server {

    static void main() {
        MinecraftServer server = MinecraftServer.init(new Auth.Velocity("my_secret"));
        InstanceContainer container = MinecraftServer.getInstanceManager().createInstanceContainer();
        container.setGenerator(unit -> unit.modifier().fillHeight(-64, 0, Block.WHITE_STAINED_GLASS));
        container.setChunkSupplier(LightingChunk::new);
        MinecraftServer.getGlobalEventHandler().addListener(AsyncPlayerConfigurationEvent.class, e -> {
            Player player = e.getPlayer();
            e.setSpawningInstance(container);
            player.scheduler().buildTask(() -> {
                player.sendPacketToViewersAndSelf(new EntitySoundEffectPacket(SoundEvent.EVENT_RAID_HORN,
                    Source.PLAYER, player.getEntityId(), 1f,1f, ThreadLocalRandom.current().nextLong()));
            }).delay(TaskSchedule.tick(20)).schedule();
        });
        server.start("localhost", 25566);
    }
}
```

Gate logs:
```
2025-11-17T22:22:01.669-0500    DEBUG   java.client.server.reader       netmc/reader.go:64      error reading packet, closing connection        {"error": "error decoding packet (type: *packet.SoundEntityPacket, id: 72, protocol: 773, direction: ClientBound, read: 21, unread: 0): EOF\nunexpected EOF"}
2025-11-17T22:22:01.669-0500    INFO    java.client.player      proxy/switch.go:318     player was kicked from server   {"name": "Foxikle", "id": "7f3208cb-05b7-4b35-819c-a58438158fbc", "server": "server1", "reason": "Internal server connection error"}
2025-11-17T22:22:01.669-0500    LEVEL(-2)       java.client.encoder     codec/encoder.go:100    encoded packet  {"context": "PacketContext:direction=ClientBound,Protocol=773,KnownPacket=true,PacketID=20,PacketType=*packet.Disconnect,Payloadlen=0", "bytes": 119}
2025-11-17T22:22:01.669-0500    DEBUG   java.client.reader      netmc/reader.go:64      error reading packet, closing connection        {"error": "error reading packet frame: error reading varint: read tcp 127.0.0.1:25565->127.0.0.1:56936: use of closed network connection"}
2025-11-17T22:22:01.669-0500    DEBUG   java.client     netmc/connection.go:290 update auto reading     {"enabled": true}
2025-11-17T22:22:01.669-0500    INFO    java.client.player      proxy/player.go:609     player has been disconnected    {"name": "Foxikle", "id": "7f3208cb-05b7-4b35-819c-a58438158fbc", "reason": "§cThe server you were on kicked you: Internal server connection error"}
2025-11-17T22:22:01.669-0500    DEBUG   java.client.server      netmc/connection.go:290 update auto reading     {"enabled": true}


```

The client is disconnected 20 ticks after connecting (as scheduled). I'm using 1.21.10. This looks similar to #597, but a different packet.

<img width="938" height="332" alt="Image" src="https://github.com/user-attachments/assets/ced77a0f-b681-41b8-9254-668f62388b66" />

Please let me know if you need any other information. You can comment here or dm/ping me in the minekube discord server. My discord id is `@foxikle`.

--- a/pkg/edition/java/proto/packet/sound.go
+++ b/pkg/edition/java/proto/packet/sound.go
@@ -131,10 +131,7 @@ func (s *SoundEntityPacket) Decode(c *proto.PacketContext, rd io.Reader) (err er
 	if s.SoundID == 0 {
 		pr.MinimalKey(&s.SoundName)
 
-		var hasFixedRange bool
-		pr.Bool(&hasFixedRange)
-
-		if hasFixedRange {
+		if pr.Ok() {
 			var fixedRange float32
 			pr.Float32(&fixedRange)
 			s.FixedRange = &fixedRange

